FiveCalls/FiveCalls/EditLocationViewController.swift

```diff
@@ -63,27 +64,12 @@ class EditLocationViewController : UIViewController, CLLocationManagerDelegate {
         delegate?.editLocationViewControllerDidCancel(self)
     }
     
-    @IBAction func submitZipCodeTapped(_ sender: Any) {
-        if validateZipCode() {
+    @IBAction func submitAddressTapped(_ sender: Any) {
+            //TODO: Does this event need to be renamed?
             Answers.logCustomEvent(withName:"Action: Used Zip Code")
```

Comment:
Good catch. I think yes, and if needed I can combine the values from the analytics for an overall report.